### PR TITLE
US5850: Content Blocks

### PR DESCRIPTION
**Pull request by samueljmello**

### CODE REVIEW

File: src/app/amount/amount.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 </div>
 <div *ngIf="store.errors.length <= 0">
   <div class="page-header page-header--border-btm">
-    <h2>Amount</h2>
+    <h2>Amount: {{ test }}</h2>
```
Comment (mjvcallibrity) [inline src/app/amount/amount.component.html]: Is there a better name for this variable?

File: config/webpack.test.js [hunk 2]
```diff
@@ -18,11 +18,12 @@ module.exports = function(options) {
     module: {
 
       preLoaders: [
+        /*
         {
           test: /\.ts$/,
           loader: 'tslint-loader',
           exclude: [helpers.root('node_modules')]
-        },
+        },*/
```
Comment (mjvcallibrity) [inline config/webpack.test.js]: Please remove comments around this

File: package.json [hunk 3]
```diff
@@ -58,6 +57,7 @@
     "file-loader": "^0.9.0",
     "html-loader": "^0.4.4",
     "html-webpack-plugin": "^2.24.0",
+    "istanbul-instrumenter-loader": "^1.1.0",
```
Comment (mjvcallibrity) [inline package.json]: Do we still need instanbul-instrumenter-loader

File: src/app/services/store.service.ts [hunk 4]
```diff
@@ -357,4 +361,21 @@ export class StoreService {
     }
   }
 
+  public dynamicData(data: string, replacement: DynamicReplace) {
+    let reg = new RegExp('\{\{ {0,}' + replacement.key + ' {0,}\}\}', 'g');
```
Comment (mjvcallibrity) [inline src/app/services/store.service.ts]: Do we need anymore `\` in this regex?

File: src/app/amount/amount.component.html (hunk 1 above)
Comment (samueljmello) [inline src/app/amount/amount.component.html]: Removed.

**Pull request merged**